src/utils.js

```diff
@@ -27,6 +27,20 @@ export const getIconImageElement = fileType => {
     return img
 }
 
+export const getIconReactComponent = fileType => {
```

Comment:
so mabybe another one of these, for the tags-related icon objects?

Reply:
added tags configuration